ddtrace.gemspec

```diff
@@ -51,8 +51,11 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'yard', '~> 0.9'
   spec.add_development_dependency 'webmock', '~> 2.0'
   spec.add_development_dependency 'builder'
-  spec.add_development_dependency 'ruby-prof'
-  spec.add_development_dependency 'sqlite3', '~> 1.3.6'
+  if RUBY_PLATFORM != 'java'
```

Comment:
Not something we have to address here, but we should consider moving these "development dependencies" entirely out of the gemspec to the `Gemfile`, as they drive our test suite mostly. (A change that would be analogous to what we did for `pry-nav`.)

Reply:
Sounds like a nice way to clean up the gemspec file 👍